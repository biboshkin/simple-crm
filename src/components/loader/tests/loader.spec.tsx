import React from "react";
import renderer from "react-test-renderer";
import { Loader } from "..";

test("render loader", () => {
  const component = renderer.create(<Loader />);
  let tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
