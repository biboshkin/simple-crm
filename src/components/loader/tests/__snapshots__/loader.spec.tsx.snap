// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render loader 1`] = `
<div
  className="loader"
>
  <div
    className="css-19d8apy"
  >
    <div
      className="css-13ej97m"
    />
    <div
      className="css-1r8me59"
    />
    <div
      className="css-73otjw"
    />
    <div
      className="css-rsevgk"
    />
    <div
      className="css-bqxwdx"
    />
    <div
      className="css-15ubpbn"
    />
  </div>
</div>
`;
